src/sdl2/hint.rs

```diff
@@ -3,6 +3,27 @@ use sys::hint as ll;
 use std::ptr;
 use libc::c_char;
 
+const VIDEO_MINIMIZE_ON_FOCUS_LOST: &'static str = "SDL_VIDEO_MINIMIZE_ON_FOCUS_LOST";
+
+/// Sets the SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS hint.
+///
+/// # Default
+/// By default a SDL_Window is minimized if it loses key focus when in fullscreen mode.
+///
+/// # Example
+/// ```no_run
+/// sdl2::hint::video_minimize_on_focus_lost(false);
+/// ```
+///
+/// * `value`:
+///    - `true`: do minimize the SDL_Window if it loses key focus when in fullscreen mode
+///    - `false`: do not minimize the SDL_Window if it loses key focus when in fullscreen mode
```

Comment:
You don't have to repeat yourself or enumerate every value like that, this would have been enough : 

```rust
/// * `value`: `true` to enable minimizing of the Window if it loses key focus when in fullscreen mode , `false` to disable this feature. Defaults to X
```

If it has a default value (and it is explicitly said in the SDL2 documentation), it would be nice to include what is the default value as well.

By the way, you don't have to say `SDL_Window` instead of `Window` when the equivalence is this obvious.